### PR TITLE
Release 1.12.3

### PR DESCRIPTION
**Pull request by rom1504bot**
Triggered on behalf of extremeheat in <a href="https://github.com/PrismarineJS/prismarine-item/pull/99#issuecomment-1614764495">this comment</a>.

<em>Note: Changes to the PR maybe needed to remove commits unrelated to library usage.</em>
<hr/>🤖 I'm a bot. You can rename this PR or run <code>/makerelease [version]</code> again to change the version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,8 +93,10 @@ See https://minecraft.gamepedia.com/Anvil_mechanics#Anvil_Uses
 
 A getter for abstracting the underlying nbt, get's the mob name from a spawn egg Item. e.g. a zombie spawn egg on 1.8 will return `Zombie`
 
-
 ## History
+### 1.12.3
```
Comment (rom1504) [inline README.md:97]: Space missing after history

Comment (rom1504) [reply README.md:97]: I mean new line

Comment (extremeheat) [reply README.md:99]: Fixed

File: README.md [hunk 2]
```diff
@@ -96,6 +96,10 @@ A getter for abstracting the underlying nbt, get's the mob name from a spawn egg
 
 ## History
 
+### 1.12.3
+* [add commands workflow (#99)](https://github.com/PrismarineJS/prismarine-item/commit/a4f5d593be8a80fbe539369efe2d1127c9785583) (thanks @extremeheat)
+* [Update to node 18.0.0 (#98)](https://github.com/PrismarineJS/prismarine-item/commit/52098ca2b0a9eafa6bcde3cdfb3c2315bab258a5) (thanks @Romain Beaumont)
```
Comment (rom1504) [inline README.md:101]: A detail but wonder if it's possible to replace the name by the pseudo in the thanks, so that @ is actually working

Comment (extremeheat) [reply README.md:101]: yeah, the action can switch to using the Github API for this (currently it's looking at local git commit logs which don't store github usernames)

Comment (extremeheat) [reply README.md]: fixed

**Pull request merged**